maxim/logger/livekit/agent_activity.py

```
@@ -2,23 +2,27 @@
 import inspect
 import time
 import traceback
+from typing import Optional
```

Ruff (0.12.2)

`typing.Optional` imported but unused

Remove unused import: `typing.Optional`

(F401)